geode-management/src/main/java/org/apache/geode/management/runtime/RestoreRedundancyResults.java

```diff
@@ -18,28 +18,28 @@
 import java.util.List;
 import java.util.Map;
 
+import org.apache.geode.annotations.Experimental;
+
 /**
  * A class to collect the results of restore redundancy operations for one or more regions and
  * determine the success of failure of the operation.
  */
+@Experimental
 public interface RestoreRedundancyResults extends OperationResult {
 
   /**
    * {@link #SUCCESS} is defined as every included region having fully satisfied redundancy.
    * {@link #FAILURE} is defined as at least one region that is configured to have redundant copies
    * having fewer than its configured number of redundant copies.
-   * {@link #ERROR} is for cases when the restore redundancy operation was unable to begin or threw
-   * an exception.
    */
   enum Status {
     SUCCESS,
-    FAILURE,
-    ERROR
```

Comment:
@DonalEvans Please take a look at this change. We get rid of the ERROR status here because we found it's never used. No one is able to set the status to that state, and the getter of it would only return either SUCCESS or FAILURE. Even on develop branch, this state seems unreachable.

Reply:
You're correct, this was originally intended to be a way to capture the case that an exception was thrown during the  `doRestoreRedundancy()` method in `RestoreRedundancyOperationImpl`, but in order to keep the implementation as similar to that found in `RebalanceOperationImpl` as possible, this was not implemented. It should be fine to remove it.